src/plugins/controls/public/range_slider/range_slider_reducers.ts

```diff
@@ -24,7 +24,15 @@ export const rangeSliderReducers = {
     state: WritableDraft<RangeSliderReduxState>,
     action: PayloadAction<RangeValue>
   ) => {
-    state.explicitInput.value = action.payload;
+    const [minSelection, maxSelection]: RangeValue = action.payload;
+    if (
+      minSelection === String(state.componentState.min) &&
+      maxSelection === String(state.componentState.max)
+    ) {
+      state.explicitInput.value = undefined;
+    } else {
+      state.explicitInput.value = action.payload;
+    }
```

Comment:
This fixes a bug where manually selecting the `min`/`max` values would not clear the unsaved changes - note that this bug was present before this PR, I just figured I'd quickly fix it here :+1:  

| Before | After |
|--------|--------|
| Cell | Cell |